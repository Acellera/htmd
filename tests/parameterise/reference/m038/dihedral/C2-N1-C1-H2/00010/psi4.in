set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 0.102908	 -0.061840	 -0.297136
N	 -0.626943	 -0.667432	 -0.000092
C	 -1.979713	 -0.158776	 0.000050
C	 -0.303657	 -1.926657	 0.382011
H	 -2.520245	 -0.492427	 -0.879443
H	 -1.934741	 0.918111	 -0.004428
H	 -2.517790	 -0.485056	 0.883818
C	 1.175299	 -2.248285	 0.303069
O	 -1.114312	 -2.729269	 0.754659
H	 1.746452	 -1.582314	 0.942302
H	 1.328421	 -3.268182	 0.622537
H	 1.539205	 -2.131733	 -0.712965

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		3 2 4 9
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "12  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
