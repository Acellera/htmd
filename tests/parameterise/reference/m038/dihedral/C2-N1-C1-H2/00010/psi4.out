    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:26:42 2016

    Process ID:  27836
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 0.102908	 -0.061840	 -0.297136
N	 -0.626943	 -0.667432	 -0.000092
C	 -1.979713	 -0.158776	 0.000050
C	 -0.303657	 -1.926657	 0.382011
H	 -2.520245	 -0.492427	 -0.879443
H	 -1.934741	 0.918111	 -0.004428
H	 -2.517790	 -0.485056	 0.883818
C	 1.175299	 -2.248285	 0.303069
O	 -1.114312	 -2.729269	 0.754659
H	 1.746452	 -1.582314	 0.942302
H	 1.328421	 -3.268182	 0.622537
H	 1.539205	 -2.131733	 -0.712965

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		3 2 4 9
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "12  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-3-2-3.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:26:43 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H          0.680199616806     1.473545659107    -0.582546192832     1.007825032070
           N         -0.049651383194     0.867953659107    -0.285502192832    14.003074004780
           C         -1.402421383194     1.376609659107    -0.285360192832    12.000000000000
           C          0.273634616806    -0.391271340893     0.096600807168    12.000000000000
           H         -1.942953383194     1.042958659107    -1.164853192832     1.007825032070
           H         -1.357449383194     2.453496659107    -0.289838192832     1.007825032070
           H         -1.940498383194     1.050329659107     0.598407807168     1.007825032070
           C          1.752590616806    -0.712899340893     0.017658807168    12.000000000000
           O         -0.537020383194    -1.193883340893     0.469248807168    15.994914619560
           H          2.323743616806    -0.046928340893     0.656891807168     1.007825032070
           H          1.905712616806    -1.732796340893     0.337126807168     1.007825032070
           H          2.116496616806    -0.596347340893    -0.998375192832     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.33840  B =      0.12948  C =      0.09701 [cm^-1]
  Rotational constants: A =  10145.00556  B =   3881.86029  C =   2908.40478 [MHz]
  Nuclear repulsion =  181.146741632724485

  Charge       = 0
  Multiplicity = 1
  Electrons    = 40
  Nalpha       = 20
  Nbeta        = 20

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 44
    Number of basis function: 89
    Number of Cartesian functions: 89
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         89      89       0       0       0       0
   -------------------------------------------------------
    Total      89      89      20      20      20       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 183
    Number of basis function: 580
    Number of Cartesian functions: 580
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 3.6472035281E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -191.08322897420416   -1.91083e+02   8.58265e-02 
   @DF-RHF iter   2:  -193.02389927593569   -1.94067e+00   5.90485e-02 DIIS
   @DF-RHF iter   3:  -219.14650347118018   -2.61226e+01   6.52183e-02 DIIS
   @DF-RHF iter   4:  -242.45599650560789   -2.33095e+01   2.98313e-02 DIIS
   @DF-RHF iter   5:  -245.71142350339974   -3.25543e+00   1.60214e-02 DIIS
   @DF-RHF iter   6:  -246.83250861975793   -1.12109e+00   5.82187e-03 DIIS
   @DF-RHF iter   7:  -246.98601424618627   -1.53506e-01   1.71834e-03 DIIS
   @DF-RHF iter   8:  -247.00309709214540   -1.70828e-02   4.89751e-04 DIIS
   @DF-RHF iter   9:  -247.00488672618130   -1.78963e-03   2.03649e-04 DIIS
   @DF-RHF iter  10:  -247.00526357409913   -3.76848e-04   7.00571e-05 DIIS
   @DF-RHF iter  11:  -247.00533615969650   -7.25856e-05   1.80388e-05 DIIS
   @DF-RHF iter  12:  -247.00534052556824   -4.36587e-06   5.10234e-06 DIIS
   @DF-RHF iter  13:  -247.00534069814537   -1.72577e-07   3.23424e-06 DIIS
   @DF-RHF iter  14:  -247.00534075146581   -5.33204e-08   5.61924e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.516383     2A    -15.581027     3A    -11.341795  
       4A    -11.256261     5A    -11.229095     6A     -1.383472  
       7A     -1.228197     8A     -1.015433     9A     -0.932461  
      10A     -0.786016    11A     -0.688423    12A     -0.647408  
      13A     -0.633010    14A     -0.585074    15A     -0.579897  
      16A     -0.556281    17A     -0.546576    18A     -0.504627  
      19A     -0.412228    20A     -0.384847  

    Virtual:                                                              

      21A      0.210290    22A      0.216740    23A      0.265231  
      24A      0.277710    25A      0.308827    26A      0.316340  
      27A      0.332934    28A      0.358274    29A      0.375314  
      30A      0.425580    31A      0.496958    32A      0.584886  
      33A      0.720692    34A      0.741255    35A      0.754386  
      36A      0.768893    37A      0.810775    38A      0.830706  
      39A      0.879788    40A      0.945083    41A      0.985929  
      42A      1.027556    43A      1.053639    44A      1.081789  
      45A      1.117777    46A      1.143040    47A      1.150022  
      48A      1.187556    49A      1.199936    50A      1.212668  
      51A      1.223169    52A      1.241818    53A      1.307684  
      54A      1.355971    55A      1.390147    56A      1.598859  
      57A      1.617968    58A      1.643576    59A      1.679544  
      60A      1.712930    61A      1.798497    62A      1.929120  
      63A      2.045936    64A      2.115614    65A      2.141306  
      66A      2.210069    67A      2.214768    68A      2.262446  
      69A      2.336297    70A      2.364924    71A      2.437048  
      72A      2.481694    73A      2.543893    74A      2.556229  
      75A      2.654782    76A      2.699310    77A      2.787197  
      78A      2.805439    79A      2.902491    80A      3.006065  
      81A      3.046689    82A      3.282546    83A      3.377201  
      84A      3.554416    85A      4.357868    86A      4.471332  
      87A      4.643480    88A      4.692454    89A      4.904499  

    Final Occupation by Irrep:
              A 
    DOCC [    20 ]

  Energy converged.

  @DF-RHF Final Energy:  -247.00534075146581

   => Energetics <=

    Nuclear Repulsion Energy =            181.1467416327244848
    One-Electron Energy =                -694.6870504795397210
    Two-Electron Energy =                 266.5349680953491998
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -247.0053407514660648

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     1.6712      Y:     3.4081      Z:    -1.3499

  Electronic Dipole Moment: (a.u.)
     X:    -0.8582      Y:    -2.1324      Z:     0.8413

  Dipole Moment: (a.u.)
     X:     0.8130      Y:     1.2757      Z:    -0.5085     Total:     1.5959

  Dipole Moment: (Debye)
     X:     2.0664      Y:     3.2425      Z:    -1.2925     Total:     4.0564


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-3-2-3.cx1.hpc.ic.ac.uk at Tue Jul  5 16:26:47 2016
Module time:
	user time   =       4.49 seconds =       0.07 minutes
	system time =       0.04 seconds =       0.00 minutes
	total time  =          4 seconds =       0.07 minutes
Total time:
	user time   =       4.49 seconds =       0.07 minutes
	system time =       0.04 seconds =       0.00 minutes
	total time  =          4 seconds =       0.07 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     1.6712      Y:     3.4081      Z:    -1.3499

  Electronic Dipole Moment: (a.u.)
     X:    -0.8582      Y:    -2.1324      Z:     0.8413

  Dipole Moment: (a.u.)
     X:     0.8130      Y:     1.2757      Z:    -0.5085     Total:     1.5959

  Dipole Moment: (Debye)
     X:     2.0664      Y:     3.2425      Z:    -1.2925     Total:     4.0564

  Quadrupole Moment: (Debye Ang)
    XX:   -28.2883     YY:   -31.1877     ZZ:   -31.2076
    XY:    -2.7849     XZ:     0.4248     YZ:     0.4935

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     1.9396     YY:    -0.9598     ZZ:    -0.9797
    XY:    -2.7849     XZ:     0.4248     YZ:     0.4935

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.30958  0.30958  0.00000  0.38085
       2     N     3.89283  3.89283  0.00000 -0.78566
       3     C     3.13361  3.13361  0.00000 -0.26722
       4     C     2.63119  2.63119  0.00000  0.73761
       5     H     0.41178  0.41178  0.00000  0.17643
       6     H     0.41762  0.41762  0.00000  0.16476
       7     H     0.39848  0.39848  0.00000  0.20305
       8     C     3.29562  3.29562  0.00000 -0.59124
       9     O     4.30097  4.30097  0.00000 -0.60194
      10     H     0.40814  0.40814  0.00000  0.18373
      11     H     0.39197  0.39197  0.00000  0.21606
      12     H     0.40821  0.40821  0.00000  0.18359

   Total alpha = 20.00000, Total beta = 20.00000, Total charge = -0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
